conf/bluebear.config

```diff
@@ -0,0 +1,40 @@
+def slurm_job_account = System.getenv("SLURM_JOB_ACCOUNT") ?: null
+
+params {
+    // Profile config
+    config_profile_description = 'BlueBEAR cluster profile'
+    config_profile_contact = 'Alex Lyttle (a.j.lyttle@bham.ac.uk)'
+    config_profile_url = 'https://docs.bear.bham.ac.uk/'
+
+    // Maximum resources available
+    max_memory = 512.GB
+    max_cpus = 112
+    max_time = 10.d
+
+    // Schema validation
+    validationSchemaIgnoreParams = 'genomes,igenomes_base,project'
+
+    // Cluster-specific params
+    project = slurm_job_account
```

Comment:
Having non-pipeline parameters here may fail some (and increasingly more pipelines).

A better approach is to tell users to set a bash environment variable with the information needed, and yse that information (in this case down in your clusterOptions).

For example, see https://github.com/nf-core/configs/blob/2900904b814873e911218c60972c15db7e119341/conf/pawsey_setonix.config#L26

Reply:
Thanks, that sounds good. In fact `System.getenv("SLURM_JOB_ACCOUNT")` could be substituted there, since it is unlikely someone would want individual processes to be submitted under a different account.